### ISSUE TITLE
Y24-055 New plate manifest type in SS to reduce the number of incorrect tags

### ISSUE THREAD

**Issue by SujitDey2022** [sanger/sequencescape#4083]
**User story**
As an SSR, I would like there to be a new library plate manifest type that allows the user to select the tag group used in the same manner as the Chromium library plate manifest - with a drop down menu for selecting the plate and wells, rather than having to type out the oligo sequences, in order to avoid typos in the oligo sequence. 

N.B. We can't use the Chromium library plate manifest, because it only works with a particular type of single index 10x tag set in which there are 4 tags per well.

**Who are the primary contacts for this story**
Richard Clark
Conor Parks

**Who is the nominated tester for UAT**
Richard Clark
Conor Parks

**Acceptance criteria**
- [x] A new manifest type is available in SS, called "Library plate - dual index tag plate".  
- [x] Above manifest type allows users to add sample tag information via a drop down menu, like the chromium type manifest.
- [x] The existing manifest type "Library type" is renamed to "Library plate - custom tags".
- [x] In the new manifest type, only tag groups relevant to this manifest type are listed in the dropdown (TBC - how to identify these - is it all dual index ones?)
- [x] In the existing Chromium library plate manifest type, only tag plates relevant to this manifest type are listed in the dropdown - these are single index 10x ones that contain 4 tags per well. It currently filters the list using 'adapter type = Chromium', but this is too permissive because that includes Chromium dual index tag sets. Need to investigate:
    - Whether to create more fine-grained adapter types and update the data (is adapter type used for anything else?)
    - Whether to create some other way to filter the tag groups (UPDATE: spin out story https://github.com/sanger/sequencescape/issues/4304)

**Dependencies**
This will be used for dual index tag sets, where i7 and i5 oligos are together in the same well. At time of writing, i7 and i5 tag sets are created separately in Sequencescape, and there is no link between two related sets (apart from the name). 

We could do a quick version where users select the i7 and i5 tag groups separately, however users would prefer to only have to select the tag set once.

This story is blocked by the following research story, to figure out how to implement this:
- https://github.com/sanger/sequencescape/issues/4160
- #4237

**Comment by SujitDey2022** [comment]
@Skrich1999 this is the new story

**Comment by harrietc52** [comment]
# Deployment

## Deploy to UAT
```
cat RELEASE
20240902163005
cat REPO
/tag/v14.41.0-beta.1
cat REVISION
f81884c1c544a950898d2e5a16c66515750c2c94
```
